src/app/pages/AccountDetailsPage/hook.ts

```diff
@@ -11,15 +11,66 @@ import { NUMBER_OF_ITEMS_ON_SEPARATE_PAGE } from '../../config'
 import { SearchScope } from '../../../types/searchScope'
 import { EventFilterMode } from '../../components/RuntimeEvents/EventListFilterSwitch'
 import { useTransformToOasisAddress } from '../../hooks/useTransformToOasisAddress'
+import { useEffect, useState } from 'react'
+import { RpcUtils } from '../../utils/rpc-utils'
 
 export const useAccount = (scope: SearchScope, address: string) => {
+  const [accountBalance, setAccountBalance] = useState<string | null>(null)
+
   const { network, layer } = scope
   if (layer === Layer.consensus) {
     // There can be no ERC-20 or ERC-721 tokens on consensus
     throw AppErrors.UnsupportedLayer
   }
   const query = useGetRuntimeAccountsAddress(network, layer, address)
-  const account = query.data?.data
+  const runtimeAccount = query.data?.data
+
+  // TODO: Remove after account balances on Nexus are in sync with the node
+  useEffect(() => {
```

Comment:
This works in https://a81a9fac.oasis-explorer.pages.dev/testnet/sapphire/address/0x2B95834655499B3C3817f379e1C8E132baa965F4
but not in https://a81a9fac.oasis-explorer.pages.dev/search?q=0x2B95834655499B3C3817f379e1C8E132baa965F4

Maybe move up into useGetRuntimeAccountsAddress